project/src/main/webapp/script.js

```diff
@@ -282,4 +282,36 @@ function createProductElement(product) {
   productElement.appendChild(linebreak);
 
   return productElement;
+}
+
+function loadReferenceImages(){
+
+  fetch("/view-reference-image").then(response => response.json()).then((referenceImages) => {
+
+    console.log(referenceImages);
+    const referenceImageElement = document.getElementById("reference-image-list");
+    referenceImageElement.innerHTML = "";
+    
+ 
+    referenceImages.forEach((referenceImage) => {  
+      referenceImageElement.appendChild(createProductElement(referenceImage));
+      referenceImageElement.appendChild(document.createElement("br"));
+    })
+  });
+}
+
+
+function createReferenceImageElement(referenceImage) {
```

Comment:
I don't see this used anywhere yet. If you don't need it anymore, I'd delete it (but obviously leave it if you're going to use it soon).